honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+    """Base class for the sets assigned to Faces."""
+
+    # TODO: What to use for default? Void or None?
+    exterior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A radiance modifier object for faces with an Outdoors boundary '
+                        'condition.'
+        )
+    # TODO: What to use for default? Void or None?
+    interior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A radiance modifier object for faces with a boundary condition '
+                        'other than Outdoors.'
+        )
+
+
+class WallModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Walls."""
+
+    type: constr(regex='^WallModifierSet$') = 'WallModifierSet'
+
+
+class FloorModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Floors."""
+
+    type: constr(regex='^FloorModifierSet$') = 'FloorModifierSet'
+
+
+class RoofCeilingModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's roofs."""
+
+    type: constr(regex='^RoofCeilingModifierSet$') = 'RoofCeilingModifierSet'
+
+
+class ShadeModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Shade."""
+
+    type: constr(regex='^ShadeModifierSet$') = 'ShadeModifierSet'
+
+
+class ApertureModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Apertures."""
+
+    type: constr(regex='^ApertureModifierSet$') = 'ApertureModifierSet'
+
+    window_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, '
+                        'and Wall parent Face.'
+        )
+
+    interior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for apertures with a Surface '
+                        'boundary condition.'
+        )
+
+    skylight_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, and a '
+                        'RoofCeiling or Floor face type for their parent face.'
+        )
+
+    operable_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for apertures with an Outdoors boundary '
+                        'condition and a True is_operable property.'
+        )
+
+
+class DoorModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Doors."""
+
+    type: constr(regex='^DoorModifierSet$') = 'DoorModifierSet'
+
+    exterior_modifier: ModifierBase = Field(
```

Comment:
You don't need to have `exterior_modifier` and `interior_modifier` here since you already get them from `BaseModifierSet`

Reply:
I was wondering if it was worth overriding this since there's a different docstring for interior/exterior modifiers in Doors that specifies it's for a 'door' object. But I can delete it since it's minor.